templates/index.php

```diff
@@ -20,6 +20,10 @@
 }
 ?>
 
+<div id="user-displayname"
+     style="display: none"><?php p(\OCP\User::getDisplayName(\OCP\User::getUser())); ?></div>
+<div id="user-email"
+     style="display: none"><?php p(\OCP\Config::getUserValue(\OCP\User::getUser(), 'settings', 'email', '')); ?></div>
```

Comment:
@DeepDiver1975 is there a better way of retrieving those settings? Something like an ajax endpoint for apps would be helpful.